### ISSUE TITLE
Support import of KML

### ISSUE THREAD

**Issue by CWGSM3V0** [hardillb/node-red-node-geofence#8]
Like this add-on but wish I could import KML so I can use another tool to draw out geofence and then import into node-red; without satellite view, my property lines are impossible to fence out otherwise and it's best-guess then.

**Comment by hardillb** [comment]
A pull requests will be looked at favourably.

But if you swap geofence.html for geofence-nomap.html you can always enter a coordinates for the North West and South East corners of a square or the centre of a circle and it's radius.

It should also be simple enough to extend this to include a list of points to make up a arbitrary shape.

**Comment by CWGSM3V0** [comment]
I ended up cheating :) I found an online tool to generate KML for my location and modified the JSON in node-red; it looks accurate to me based on the map display - just need to test. Unfortunately, I don't have the necessary skills (weekend warrior) to even attempt to add the support but figured it never hurts to ask. As an example, in my case, I chose a square/box as it's accurate enough for the property line:

[{"id":"9a03a64c.d1b7e8","type":"geofence","z":"3243b776.4914d8","name":"","mode":"polyline","inside":"true","rad":0,"points":[{"latitude":x,"longitude":y},{"latitude":x,"longitude":y},{"latitude":x,"longitude":y},{"latitude":x,"longitude":y}],"centre":{},"x":269.16668701171875,"y":304,"wires":[[]]}]

Not 100% sure what the center-x-y is for but assuming it'll be fine.
